### PR TITLE
[ws-daemon] Check blkio throttle exists

### PR DESCRIPTION
**Pull request by aledbf**
## Description

Remove useless log https://cloudlogging.app.goo.gl/YcGHidiz6YFSa8T58

## Release Notes
```release-note
[ws-daemon] Check blkio throttle exists
```

### CODE REVIEW

**Review by Furisto — DISMISSED** [review]

**Review by csweichel — COMMENTED** [review]
nits

File: components/ws-daemon/pkg/cgroup/plugin_iolimit_v1.go [hunk 1]
```diff
@@ -94,11 +95,17 @@ func (c *IOLimiterV1) Apply(ctx context.Context, basePath, cgroupPath string) er
 		return lines
 	}
 
-	writeLimit := func(fn string, content []string) error {
+	writeLimit := func(limitPath string, content []string) error {
 		for _, l := range content {
-			err := os.WriteFile(fn, []byte(l), 0644)
+			_, err := os.Stat(limitPath)
+			if errors.Is(err, os.ErrNotExist) {
+				log.WithError(err).WithField("path", limitPath).Debug("file does not exist")
```
[the remark this answers is not in the document]
Comment (aledbf) [reply components/ws-daemon/pkg/cgroup/plugin_iolimit_v1.go:102]: fixed

File: components/ws-daemon/pkg/cgroup/plugin_iolimit_v1.go [hunk 2]
```diff
@@ -94,11 +95,17 @@ func (c *IOLimiterV1) Apply(ctx context.Context, basePath, cgroupPath string) er
...
+				continue
+			}
+
+			err = os.WriteFile(limitPath, []byte(l), 0644)
 			if err != nil {
-				log.WithError(err).WithField("fn", fn).WithField("line", l).Warn("cannot write limit")
+				log.WithError(err).WithField("fn", limitPath).WithField("line", l).Warn("cannot write limit")
```
[the remark this answers is not in the document]
Comment (aledbf) [reply components/ws-daemon/pkg/cgroup/plugin_iolimit_v1.go:108]: fixed

**Review by csweichel — APPROVED** [review]

**Comment by kylos101** [comment]
Added to our changelog

**Pull request merged**